### PR TITLE
fixed formating; add tests

### PR DESCRIPTION
**Pull request by wanderer**
with 2 spaces the formating now looks like 
```
 (; ;)
  (module
    (func
      (param i32))
    (func
      (param i32)))
```
browing from clojure
also added tests from #8

### CODE REVIEW

**Comment by wanderer** [comment]
i would like to get rid of ' (; ;)' but its proven to be a toughy

**Comment by drom** [comment]
@wanderer `script` removal is not urgent. Let's focus on the functional bugs first.

**Comment by wanderer** [comment]
well its removed now, but if there maybe a better way todo it

**Comment by wanderer** [comment]
next up; fixing literals

**Comment by wanderer** [comment]
next up; fixing export

**Comment by wanderer** [comment]
formating now looks like
```
(i32.add
  (get_local $x)
  (get_local $y))

```
and 
```
(module
  (func
     (param i32))
   (func
     (param i32)))
```

File: bin/generator.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ var compositeName = {
     relop: 'node.type + \'.\' + node.operator',
     cvtop: 'node.type + \'.\' + node.operator + \'/\' + node.type1',
     const: 'node.type + \'.const \' + node.init',
-    identifier: '\'$\' + node.name',
+    identifier: '\'$\' + node.id',
```
Comment (drom) [inline bin/generator.js]: @wanderer it is `identifier.name` as i remember. `id` is an Object branch key

File: bin/generator.js [hunk 2]
```diff
@@ -80,19 +81,31 @@ function funcObject (obj) {
...
+                exprGen[n.kind](n);
+              })
+            }else{
+              exprGen[node.kind](node);
+            }
+
+            // trims leadin newline
+            if (space) {
+              res = res.slice(1)
+            }
```
Comment (drom) [inline bin/generator.js]: It is a hacky code. There should be a better way, but OK for now.

File: bin/generator.js (hunk 1 above)
Comment (wanderer) [inline bin/generator.js]: yep, i must have an old version of wast-parser

Comment (wanderer) [inline bin/generator.js]: ok fixed

**Pull request merged**